### PR TITLE
Geo reviewers only access for geo and beta version message

### LINKED ISSUE #3269: GEO: Enable geo page only for reviewers with disclaimer text
- [ ] Geo Page must be access only by reviewers, redirect to cycle home when user is not reviewer (use Users.isReviewer)
- [ ] Add to toolbar message : FRA GEO - Beta version (see screenshot below) (use same style as Assessment label in header, maybe a couple of pixel smaller)

Image

### PR DESCRIPTION
**Pull request by yaguzmang**
- Added access for reviewers only to Geo
- Added a Beta Version message header

image

### CODE REVIEW

File: src/client/components/PageLayout/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -53,6 +54,11 @@ const Toolbar: React.FC = () => {
         />
       </div>
 
+      {isInGeoPage && (
+        <MediaQuery minWidth={Breakpoints.tabletPortrait}>
```
Comment (sorja) [inline src/client/components/PageLayout/Toolbar/Toolbar.tsx:58]: Is `MediaQuery` necessary here

File: src/client/pages/Geo/Geo.tsx [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
 import Loading from 'client/components/Loading'
 
 const GeoMapWrapper = React.lazy(() => import('./GeoMapWrapper'))
 
 const Geo: React.FC = () => {
+  const user = useUser()
+  const countryIso = useCountryIso()
+  const cycle = useCycle()
+
+  const isReviewer = Users.isReviewer(user, countryIso, cycle)
```
Comment (sorja) [inline src/client/pages/Geo/Geo.tsx:20]: Should work also for administrators

File: src/client/components/PageLayout/Toolbar/Toolbar.tsx (hunk 1 above)
Comment (yaguzmang) [reply src/client/components/PageLayout/Toolbar/Toolbar.tsx:58]: Yes. It wouldn't fit into smaller screens even If I remove the spacing.  


https://github.com/openforis/fra-platform/assets/41337901/b5c5e3aa-b6d2-48bb-8ca2-ca799a40ae14

File: src/client/pages/Geo/Geo.tsx (hunk 2 above)
Comment (yaguzmang) [reply src/client/pages/Geo/Geo.tsx:20]: Added. Thanks!

**Review by minotogna — APPROVED** [review]

**Pull request merged**